ci/release/update-version.sh

```diff
@@ -50,6 +46,7 @@ for DEP in "${DEPENDENCIES[@]}"; do
   for FILE in dependencies.yaml conda/environments/*.yaml; do
     sed_runner "/-.* ${DEP}\(-cu[[:digit:]]\{2\}\)\{0,1\}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*,>=0.0.0a0/g" "${FILE}"
   done
+  # shellcheck disable=SC2043
   for FILE in python/pyproject.toml; do
```

Comment:
shellcheck doesn't like to iterate over loops of a single file, but this seems fine (and simpler than refactoring)